pkg/controller/dynamic/dynamic_controller_integration_test.go

```diff
@@ -249,7 +251,10 @@ func validateCreate(ctx context.Context, t *testing.T, testContext testrunner.Te
 
 	// Check that an "Updating" event was recorded, indicating that the
 	// controller tried to update the resource at all.
-	testcontroller.AssertEventRecordedforUnstruct(t, kubeClient, reconciledUnstruct, k8s.Updating)
+	// TODO(acpana): figure out if we want to expose Updating event for direct resources
```

Comment:
Good call.  Can you open an issue for 1.119?  We should be creating the same events, and we should have test coverage.